package/scripts/common/quarto

```diff
@@ -27,7 +27,7 @@ if [ -f "$DEV_PATH" ]; then
     QUARTO_TARGET=$DEV_PATH
   fi
   export QUARTO_BIN_PATH=$SCRIPT_PATH
-  export QUARTO_SHARE_PATH="`cd "$SCRIPT_PATH/../../../src/resources/";pwd`"
+  export QUARTO_SHARE_DIR="`cd "$SCRIPT_PATH/../../../src/resources/";pwd`"
```

Comment:
See previous note about continuing to respect the previous `QUARTO_SHARE_PATH` variable for the sake of the IDE.